packages/sourcecred/src/ui/components/AccountOverview.js

```diff
@@ -126,6 +166,25 @@ export const AccountOverview = ({
               AccountRow(a, currencySuffix, decimalsToDisplay)
             )}
           </TableBody>
+
+          <TableFooter>
+            <TableRow>
+              <TablePagination
+                rowsPerPageOptions={PAGINATION_OPTIONS}
+                className={classes.paginator}
+                colSpan={4}
+                count={tsAccounts.length}
+                rowsPerPage={tsAccounts.rowsPerPage}
+                page={tsAccounts.pageIndex}
```

Comment:
nit: could we possibly add an option to display 25 rows or even 10 rows per page?

Reply:
Sorry, I didn't catch this comment. the options are 10, 25, 50 items/page.